spec/services/json_webhook_service_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
```

Comment:
Do we need rails_helper? can we just use spec_helper?

Reply:
gives this error when spec_helper is used 
```ruby
  uninitialized constant JsonWebhookService
```